### PR TITLE
fix(docs): replace older trpc links with new ones

### LINKED ISSUE #1298: bug: Incorrect anchor link in src/server/api/trpc.ts comments
### Provide environment information

n/a 

### Describe the bug

There is a link with an invalid anchor in all 4 versions of the `src/server/api/trpc.ts` template file [1](https://github.com/t3-oss/create-t3-app/blob/next/cli/template/extras/src/server/api/trpc/base.ts#L30) [2](https://github.com/t3-oss/create-t3-app/blob/next/cli/template/extras/src/server/api/trpc/with-auth-prisma.ts#L35) [3](https://github.com/t3-oss/create-t3-app/blob/next/cli/template/extras/src/server/api/trpc/with-auth.ts#L34) [4](https://github.com/t3-oss/create-t3-app/blob/next/cli/template/extras/src/server/api/trpc/with-prisma.ts#L31)
The bad link is - https://create.t3.gg/en/usage/trpc#-servertrpccontextts

This link also in two pages in the docs.
* [folder structure page](https://create.t3.gg/en/folder-structure#srcserverapitrpcts) - the first `tRPC usage` link
* [usage/prisma](https://create.t3.gg/en/usage/prisma#prisma-client) the `context` link

### Reproduction repo

https://github.com/t3-oss/create-t3-app/blob/next/cli/template/extras/src/server/api/trpc/base.ts#L30

### To reproduce

n/a

### Additional information

I *think* the link should be - https://create.t3.gg/en/usage/trpc#-serverapitrpcts 

Can toss a quick PR up if you can confirm this is the correct anchor to link to.

### PR DESCRIPTION
**Pull request by PiotrekPKP**
Closes #1298 

## ✅ Checklist

- [x] I have followed every step in the [contributing guide](https://github.com/t3-oss/create-t3-app/blob/main/CONTRIBUTING.md) (updated 2022-10-06).
- [x] The PR title follows the convention we established [conventional-commit](https://www.conventionalcommits.org/en/v1.0.0/)
- [ ] I performed a functional test on my final commit

---

## Changelog

Replaced incorrect links to trpc usage page with working urls.

---

💯

### CODE REVIEW

File: www/src/pages/pt/folder-structure.mdx [hunk 1]
```diff
@@ -114,16 +114,16 @@ Dependendo dos pacotes escolhidos, este roteador contém mais ou menos rotas par
 
 O arquivo `trpc.ts` é o arquivo de configuração principal para seu back-end tRPC. Aqui nós:
 
-1. Definimos o contexto usado nas solicitações tRPC. Veja [uso do tRPC](usage/trpc#-servertrpccontextts) para mais informações.
-2. Exportamos os auxiliares de procedimento. Veja [uso do tRPC](usage/trpc#-servertrpctrpcts) para mais informações.
+1. Definimos o contexto usado nas solicitações tRPC. Veja [uso do tRPC](usage/trpc#-serverapitrpcts) para mais informações.
```
Comment (Sn0wye) [inline www/src/pages/pt/folder-structure.mdx:117]:
```suggestion
1. Definimos o contexto usado nas requests tRPC. Veja [uso do tRPC](usage/trpc#-serverapitrpcts) para mais informações.
```

File: www/src/pages/pt/folder-structure.mdx [hunk 2]
```diff
@@ -114,16 +114,16 @@ Dependendo dos pacotes escolhidos, este roteador contém mais ou menos rotas par
...
+2. Exportamos os auxiliares de procedimento. Veja [uso do tRPC](usage/trpc#-serverapitrpcts) para mais informações.
```
Comment (Sn0wye) [inline www/src/pages/pt/folder-structure.mdx:118]:
```suggestion
2. Exportamos os auxiliares de procedures. Veja [uso do tRPC](usage/trpc#-serverapitrpcts) para mais informações.
```

**Review by Sn0wye — COMMENTED** [review]
Little changes, and done!

**Comment by escwxyz** [comment]
LGTM

**Review by juliusmarminge — APPROVED** [review]

**Comment by juliusmarminge** [comment]
Thanks!

**Pull request merged**